tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -35,18 +34,24 @@ sub run {
     # It may happen that the screen sharing is not available
     assert_screen [qw(with_screensharing without_screensharing)];
     if (match_has_tag 'without_screensharing') {
-        record_info 'vino missing', 'After the installation the screen sharing is not available - vino is missing and we need to install it now.';
-        send_key 'ctrl-q';
-
-        # Install the vino package which is probably the case of missing screen sharing option
-        ensure_installed 'vino';
+        if (is_sle("=15-sp4")) {
```

Comment:
I would use 15-SP4+, I don't think on next SLE version it will go back to vino

Reply:
Thanks, I have updated the code to 15-SP4+.